src/modules/entity/domain-model/media_player.js

```diff
@@ -147,6 +151,10 @@ export default class MediaPlayer {
     this.callService('volume_down');
   }
 
+  input(input) {
```

Comment:
can you put it in alphabetical order and make the name reflect the service name: `selectSource`